Project 1/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>   
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel= "stylesheet" href= "styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="card"> 
+    <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
+<h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so intelligent.
+By Carl Zimmer</h2><h3>Nov. 30, 2018</h3>
+<figure>
+    <img src="https://static01.nyt.com/images/2018/12/04/science/04SCI-ZIMMER/merlin_147494628_a788760f-4209-40db-a0be-1305c3a661a6-articleLarge.jpg?quality=75&auto=webp&disable=upscale" alt="OctopusOnRocks">
+</figure><div class="card">
```

Comment:
If you mean for this to close the `div` you created on line 11, you want to replace this with:
```suggestion
</figure></div>
```

If you mean to have two `div`s with the classname of `card` (which would be valid and okay!), then you also need to close these `div`s at some point with a `</div>`.

Reply:
This was never resolved, so your markup is still invalid.